static/css/style.css

```diff
@@ -13,4 +13,6 @@ body {
     top: 0;
     bottom: 0;
     width: 100%;
+    height: 100%;
+    background-color: #333333;
```

Comment:
Does this have any effect except the color of the canvas before loading the base map?